packages/dialtone-css/lib/build/less/components/toast.less

```diff
@@ -119,3 +119,145 @@
+  --toast-alternate-color-background: var(--dt-color-surface-primary);
+  --toast-alternate-color-text: var(--dt-color-foreground-primary);
+  --toast-alternate-color-icon: var(--dt-color-foreground-muted);
+  --toast-alternate-font-color: var(--dt-color-foreground-tertiary);
+  --toast-alternate-padding: var(--dt-space-450);
+  --toast-alternate-font-size: var(--dt-font-size-200);
+  --toast-alternate-line-height: var(--dt-font-line-height-300);
+  --toast-alternate-border-radius: var(--dt-size-500);
+  --toast-alternate-border-color: var(--dt-color-border-subtle);
+  --toast-alternate-border-style: solid;
+  --toast-alternate-border-width: var(--dt-size-100);
+
+  z-index: var(--zi-notification);
+
+  //  Base Styles
+  //  ------------------------------------------------------------------------
+  display: flex;
+  display: block;
+  align-items: center;
+  box-sizing: border-box;
+  width: var(--dt-size-100-percent);
+  min-width: var(--dt-size-925);
+  max-width: var(--dt-size-1020);
+  padding: var(--toast-alternate-padding);
+  padding: var(--dt-space-450);
+  color: var(--toast-alternate-color-text);
+  font-size: var(--toast-alternate-font-size);
+  line-height: var(--toast-alternate-line-height);
+  word-break: normal;
+  background-color: var(--toast-alternate-color-background);
+  border: var(--toast-alternate-border-width) var(--toast-alternate-border-style) var(--toast-alternate-border-color);
+  border-radius: var(--toast-alternate-border-radius);
+  box-shadow: var(--toast-alternate-box-shadow);
+  pointer-events: all;
+
+  //  If there's a link in a toast, break the url so it wraps
+  > a {
+    word-break: break-all;
+  }
+
+  // If you want to hide and reveal the toast
+  &[aria-hidden='true'] {
+    visibility: hidden;
+    opacity: var(--dt-opacity-0);
+  }
+
+  &[aria-hidden='false'] {
+    visibility: visible;
+    opacity: var(--dt-opacity-1300);
+  }
+
+  .d-toast-layout-alternate__icon {
+    display: flex;
+    flex: 0 auto;
+    margin-right: var(--dt-space-300);
+    color: var(--toast-alternate-color-icon);
+  }
+
+  .d-notice__title {
+    color: var(--dt-color-foreground-muted);
+    font-weight: var(--dt-font-weight-normal);
+    font-size: var(--dt-font-size-100);
+    word-break: break-all;
+  }
+
+  .d-notice__actions .base-button__button {
+    padding: var(--dt-size-100);
+    color: var(--dt-action-color-foreground-muted-default);
+  }
+}
+
+//  ============================================================================
+//  $   CONTENT CONTAINER
+//  ----------------------------------------------------------------------------
+.d-toast-alternate__content {
+  display: flex;
+  flex: 1 auto;
+  flex-direction: column;
+  margin-right: var(--dt-space-500);
+  color: var(--dt-color-foreground-primary);
+  font-weight: var(--dt-font-weight-normal);
+  font-size: var(--dt-font-size-200);
+  word-break: break-all;
```

Comment:
I don't think we want to use this for all content, it will split words in half at the end of the line

someth
ing like this

the `> a` break-all that you already have should be sufficient. You can remove this one.